db/schema.rb

```diff
@@ -92,6 +92,7 @@
     t.integer  "creator_id"
     t.datetime "deleted_at"
     t.string   "slug",                                null: false
+    t.integer  "max_members"
```

Comment:
For production applications we're going to want to extract the database migration as it's own PR and we'll deploy and run the migration before we merge this PR.

@dinever if you need any assistance with that please let me know.